### PR TITLE
Add more tests for column and config

### PR DESCRIPTION
**Pull request by tpcstld**
@sb2nov PTAL

### CODE REVIEW

File: dataduct/database/tests/test_column.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+
+class TestColumn(TestCase):
+    """Tests for the Column class
+    """
+    @staticmethod
+    def test_column_construction():
+        """Tests that the constructor for the column class is working
+        """
+        column = Column(
```
Comment (sb2nov) [inline dataduct/database/tests/test_column.py]: shouldn't we be testing the parser function as most of the tests here seem to just check the default values .. or may be my brain isn't working anymore.

Comment (tpcstld) [inline dataduct/database/tests/test_column.py]: There is no parser function in the column class.

Comment (sb2nov) [inline dataduct/database/tests/test_column.py]: there is a column parsers .. check the parsers directory ?

**Comment by sb2nov** [comment]
LGTM

**Pull request merged**